client/app/containers/tests-runs/tests-runs-search/modal/search-modal.html

```diff
@@ -0,0 +1,56 @@
+<md-dialog flex="50" class="tests-runs-search__modal" aria-label="Tests runs modal" zafira-background-theme="modal" class="modal-min-width">
+    <div class="tests-runs-search__modal__wrapper">
```

Comment:
incorrect BEM using: instead of `tests-runs-search__modal__wrapper` should be like `tests-runs-search__modal-wrapper`